fairgraph/electrophysiology.py

```diff
@@ -105,22 +106,30 @@ class MultiChannelMultiTrialRecording(Trace):
     If you have a file containing only a single recording from a single channel,
     you may instead use :class:`Trace`."""
     namespace = DEFAULT_NAMESPACE
-    _path =  "/electrophysiology/multitrace/v0.1.0"  # for nexus
+    _path =  "/electrophysiology/multitrace/v0.1.1"  # for nexus
     #path = DEFAULT_NAMESPACE + "/electrophysiology/multitrace/v0.3.0"  # for nexus-int
     type = ["prov:Entity", "nsg:MultiChannelMultiTrialRecording"]
     fields = (
         Field("name", basestring, "name", required=True),
         Field("data_location", Distribution, "distribution", required=True),
-        Field("generated_by", "electrophysiology.PatchClampExperiment", "wasGeneratedBy", required=True),
-        Field("generation_metadata", "electrophysiology.QualifiedTraceGeneration", "qualifiedGeneration", required=True),
+# MULTIPLE OPTIONS
+        #Field("generated_by", "electrophysiology.ExtracellularElectrodeExperiment" or  "electrophysiology.PatchClampExperiment", #"wasGeneratedBy", required=True),
```

Comment:
Here you could put a tuple of classes `(electrophysiology.PatchClampExperiment, electrophysiology.ExtracellularElectrodeExperiment)` or you could define a base class (e.g. `ElectrophysiologyExperiment`) from which both classes could inherit.